### DISCUSSION TITLE
Is there a demo with an example implementation?

### DISCUSSION THREAD

**Discussion by k00ni** [eclipse-dataspace-protocol-base/DataspaceProtocol#36] [General]
**Is there a very basic example available how to implement the protocol?** Such as a mini-HTML file with some RDFa providing catalog info or an example JSON response of a web service.

I am still very new to all of this but at least DCAT and ODRL ([ref](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/blob/main/catalog/catalog.protocol.md#12-dcat-and-odrl-profiles)) related content is to be published. Besides, there is communication between web services/programs as mentioned [here](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/blob/main/negotiation/contract.negotiation.protocol.md#21-contract-request-message) (between Consumers and Providers).

If there is nothing available yet, are there plans to change that?

**Comment by arnoweiss** [comment]
I think there's some wrong expectation implied in this question. This protocol defines a set of HTTP endpoints that are designed without any application having to be aware of RDF. [Look at the spec](https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/), refer to the linked json-schemas in the messages and implement the http bindings.

That said, if you need a ready-to-run setup, the [Eclipse EDC MVD](https://github.com/eclipse-edc/MinimumViableDataspace) will be a good start.
